services/static-webserver/client/source/class/osparc/dashboard/Dashboard.js

```diff
@@ -47,7 +47,8 @@ qx.Class.define("osparc.dashboard.Dashboard", {
       barPosition: "top"
     });
 
-    osparc.wrapper.Plotly.getInstance().init();
+    // osparc.wrapper.Plotly.getInstance().init();
```

Comment:
you like commented code ?

Reply:
It doesn't bother me, and it makes it clear that we intentionally don't want to initialize those libraries. Since this might change, I rather leave it commented.